### ISSUE TITLE
Pulsar Functions (Python): type object 'MessageId' has no attribute 'earliest'

### ISSUE THREAD

**Issue by pouledodue** [apache/incubator-pulsar#1486]
#### Actual behavior

```
10:41:29.150 [Timer-7] ERROR org.apache.pulsar.functions.runtime.RuntimeSpawner - Function Container is dead with exception
java.lang.RuntimeException: Traceback (most recent call last):
  File "/dev/pulsar/apache-pulsar-2.0.0-streamlio-6/instances/python-instance/python_instance_main.py", line 33, in <module>
    import pulsar
  File "/dev/pulsar/apache-pulsar-2.0.0-streamlio-6/instances/python-instance/pulsar.py", line 110, in <module>pouledodue
    class MessageId:
  File "/dev/pulsar/apache-pulsar-2.0.0-streamlio-6/instances/python-instance/pulsar.py", line 116, in MessageId
    earliest = _pulsar.MessageId.earliest
AttributeError: type object 'MessageId' has no attribute 'earliest'
```

#### Steps to reproduce

[this python tutorial](http://pulsar.incubator.apache.org/docs/latest/functions/quickstart/#Writingandrunninganewfunction-5bheo)

#### System configuration
```
**Pulsar version**: 2.0.0
mac OSX El Capitan 10.11.6
tested with python 3.6.2 and 2.7.12
pulsar-client==1.19.0
```

**Comment by srkukarni** [comment]
@lucperkins

**Comment by merlimat** [comment]
`MessageId.earliest` was added since 1.19, though we don't have binaries for MacOS 10.11 since 1.20

**Comment by pouledodue** [comment]
> don't have binaries for MacOS 10.11 

will they be available?

**Comment by merlimat** [comment]
I need to check if I can get a 10.11 install working with vagrant. Then it would be easy to compile and publish the binaries. I'll keep you posted here.

**Comment by merlimat** [comment]
@pouledodue I had published 1.22.0 binaries for MacOS 10.11 https://pypi.org/project/pulsar-client/#files
